tests/patterns/python/any_qualified_with_wildcard.py

```diff
@@ -0,0 +1,16 @@
+# will match because wildcard imports in matching are acontextual, unfortunately
+# ERROR:
+bar(
```

Comment:
Should we perhaps just collect wildcard imports before any definition is made?

Reply:
I believe this to be an unrealistic scenario, and that it would be better to be generous with our matching, than miss a case. I think it would be easy, for instance, to whitelist certain kinds of definitions which it can appear before, and then accidentally forget about one.